src/app/welcome/index.njk

```diff
@@ -17,8 +17,7 @@ Create GAR | Start
 {% endblock %}
 
 {% block content %}
-<!--<p class="govuk-body-l"> <span style="background-color: #FFFF00">  {{__('covid_19_notification')}} <a href="{{__('covid_19_notification_link')}}">here</a> </span>  </p> -->
-<p class="govuk-body-l"> <span style="background-color: #66ff66"> {{__('id_card_withdrawl_text')}} <a href=https://www.gov.uk/guidance/visiting-the-uk-as-an-eu-eea-or-swiss-citizen>GOV.UK</a>. </span> </p>
+<p class="govuk-body-l"> <span style="background-color: #00ffff"> {{__('homepage_banner_text')}} <a href="https://www.gov.uk/government/consultations/advance-information-requirements-for-international-general-aviation-flights">Advance information requirements for international General Aviation flights - GOV.UK (www.gov.uk).</a>. </span> </p>
```

Comment:
@anushasivakumar do you think this  - `Advance information requirements for international General Aviation flights - GOV.UK (www.gov.uk).` - is correct for the link text having seen Sonali's email?
rather than simply `GOV.UK` as it was before?

-- Sonali's Email

The GA Consultation: Advance information requirements for international General Aviation flights has launched and will be open until 16 June 2023. Please use this link if you would like to respond: [Advance information requirements for international General Aviation flights - GOV.UK (www.gov.uk)](https://www.gov.uk/government/consultations/advance-information-requirements-for-international-general-aviation-flights).

Reply:
Hi Henry, I think it is better to have the link text as suggested by Sonali as it explains the purpose of the link better